_data/projects/PyTorch

```diff
@@ -0,0 +1,14 @@
+name: PyTorch
+desc: PyTorch is an open source machine learning library, used for applications such as computer vision and NLP
+site: https://pytorch.org/
+tags:
+- python
+- c++
+- cuda
+- machine learning
+upforgrabs:
+  name: help needed
+  link:https://github.com/pytorch/pytorch
```

Comment:
@sangy987  Change the link to something like https://github.com/pytorch/pytorch/labels/good%20first%20issue as per the label name that you will use.

Reply:
okay thanks